src/ros_filter.cpp

```diff
@@ -1018,6 +1026,7 @@ void RosFilter::loadParams()
 
         topic_subs_.push_back(
           node_->create_subscription<nav_msgs::msg::Odometry>(odom_topic,
+          rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default)),
```

Comment:
I believea ll of these subscribers have a parameter for queue size that QoS should use

Reply:
You were right! Corrected in 0423eaf. Thanks!

Reply:
I meant that `rclcpp::SystemDefaultsQoS()` doesnt specify the queue size, but the parameters for r_l will set them `odomN_queue_size: 2`, so this will ignore the parameterizations for all the subs of data

Reply:
Okay. I see what you mean. I looked at the [melodic-devel version](https://github.com/cra-ros-pkg/robot_localization/blob/melodic-devel/src/ros_filter.cpp#L1071) and tried to mirror the reading of `odom`, `pose`, `twist`, and `imu` `_queue_size`s from parameters. `SystemDefaultsQoS()` is eliminated in daba06a.

Reply:
Awesome thanks for that